odl/solvers/functional/functional.py

```diff
@@ -136,15 +136,21 @@ def convex_conj(self):
 
         The concept is also known as the Legendre transformation.
 
-        References
-        ----------
         Wikipedia article on `Convex conjugate
         <https://en.wikipedia.org/wiki/Convex_conjugate>`_.
 
         Wikipedia article on `Legendre transformation
         <https://en.wikipedia.org/wiki/Legendre_transformation>`_.
 
         For literature references see, e.g., [Lue1969]_, [Roc1970]_.
+
+        References
+        ----------
```

Comment:
This heading shouldn't have been moved. Also move the sentence "For literature ..." out of References.